editors/code/package.json

```diff
@@ -39,11 +39,13 @@
         "package": "vsce package -o clarity-lsp.vsix",
         "compile": "tsc -b",
         "build": "tsc",
+        "build:syntax": "js-yaml syntaxes/clarity.yaml > syntaxes/clarity.tmLanguage.json",
         "watch": "tsc --watch",
         "lint": "tsfmt --verify && eslint -c .eslintrc.js --ext ts ./src",
         "fix": "tsfmt -r && eslint -c .eslintrc.js --ext ts ./src --fix"
     },
     "dependencies": {
+        "js-yaml": "^3.13.1",
```

Comment:
should this be in the Dev dependencies instead?

Reply:
Moved `js-yaml` to dev dependencies.